week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(priceInPound) {
+  return "$" + priceInPound * 1.4;
+}
```

Comment:
As with my last comment, the function is now tied not only to specific currencies, but also to a particular exchange rates, as both have been hard-coded into the function. One to think about as your skills develop.